tests/test_json_schema.py

```diff
@@ -142,5 +142,9 @@ class CustomField(typesystem.Field):
         pass
 
     field = CustomField()
-    with pytest.raises(TypeError):
+    with pytest.raises(TypeError) as exc_info:
         to_json_schema(field)
+
+    message = str(exc_info.value)
+    assert message.startswith("Unsupported field type")
```

Comment:
Good call!

Can we just use `assert message == "Unsupported field type 'CustomField' passed to 'to_json_schema"` here?

Reply:
no, because the error is ```
Unsupported field type 'test_to_json_schema_invalid_field.<locals>.CustomField' passed to 'to_json_schema'
``` in full.

Reply:
no, because the error is

```
Unsupported field type 'test_to_json_schema_invalid_field.<locals>.CustomField' passed to 'to_json_schema'

```
in full.